main/index.js

```diff
@@ -49,7 +49,10 @@ app.on('quit', function(event, exitCode) {
 // initialization and is ready to create browser windows.
 app.on('ready', function() {
   globalShortcut.register('CmdOrCtrl+W',()=>{
```

Comment:
Is there should be a code that removes global shortcut when installer app is closed? What happens if you run installer with this fix, then exit it and then try to use Cmd+W in any other window?